synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1980,7 +1983,6 @@ CanvasView::create_tab_label()
 	label->show();
 
 	closebutton = manage(new Gtk::Button());
-	closebutton->set_margin_end(4);
```

Comment:
On GNOME desktop default Adwaita theme, close button is hidden slightly: https://github.com/synfig/synfig/pull/1774#discussion_r504351483

That line is not safe to be deleted. Maybe you could add a single-line comment above so other contributors know why.

Reply:
Thanks, didn’t know that 👍